### DISCUSSION TITLE
More dynamic title in the Layout

### DISCUSSION THREAD

**Discussion by PeppeL-G** [vuepress/core#480] [Q&A] [answered]
I want the title/RouterLink of the page in the upper left corner of the default layout to be more customizable. More specifically, the problem I have is that my VuePress site consists of multiple "products", for example:

```
/android (start page for Android)
/android/what-it-is
/android/how-to-use-it
/android/how-to-get-a-device
/ios (start page for iOS)
/ios/what-it-is
/ios/how-to-use-it
/ios/how-to-get-a-device
/windows (start page for Windows)
/windows/what-it-is
/windows/how-to-use-it
/windows/how-to-get-a-device
```

When the user is on a page starting with `/android`, I want the title in the upper left corner of the default layout to say *Android*, and clicking on it should lead to `/android`, no matter which `/android*`-page the user is currently viewing, and corresponding functionality for the other "products".

At the moment in the default layout, I can only use the sidebar to indicate which "product" the user is currently viewing. I rather have the NavBar indicating that in the upper left corner.

Am I alone wanting this feature, or does other want it too?

I don't think the default theme currently provides a slot for the title in the upper left corner. One solution could be to provide a slot for it. If the functionality can be implemented and added through a plugin, that could be a good solution for me too, but I would like someone to first confirm that is possible before I try to do it.

**Comment by Mister-Hope** [comment, marked as answer]
> I don't think the default theme currently provides a slot for the title in the upper left corner. One solution could be to provide a slot for it. If the functionality can be implemented and added through a plugin, that could be a good solution for me too, but I would like someone to first confirm that is possible before I try to do it.

You can use `alisas` to overide navbar component to achieve that.
